pkgs/tools/misc/coreutils/default.nix

```diff
@@ -117,6 +117,10 @@ stdenv.mkDerivation (rec {
 
   NIX_LDFLAGS = optionalString selinuxSupport "-lsepol";
   FORCE_UNSAFE_CONFIGURE = optionalString stdenv.hostPlatform.isSunOS "1";
+  NIX_CFLAGS_COMPILE =
+    # Work around a bogus warning in conjunction with musl.
+    (optionalString stdenv.hostPlatform.isMusl "-Wno-error")
+    + (optionalString stdenv.hostPlatform.isAndroid "-D__USE_FORTIFY_LEVEL=0");
```

Comment:
Uh well, maybe musl on android doesn't make sense, but perhaps at least put an extra space in there?

Generally I tend to use lists, as those behave nicer:
```nix
optional stdenv.hostPlatform.isMusl "-Wno-error"
++ optional stdenv.hostPlatform.isAndroid "-D__USE_FORTIFY_LEVEL=0"
```
but I don't mind either way.